API.md

```diff
@@ -25,7 +25,7 @@
   - [Smart objects](#smart-objects)
   - [__rerum](#rerum)
     - [History](#history)
-    - [`generatedBy` Attribution](#generatedby-attribution)
+    - [`generator` Attribution](#generator-attribution)

```

Comment:
You've replaced `generatedBy` throughout with `generator` in this API. The attribute is still called `generatedBy`, but it would be fair to call the predicate of this the Generator. The style of `generator` makes it feel like I should be able to fidn that directly in the JSON, which I can't.

Reply:
Yeah, I mirror that feeling.  I couldn't tell what was meant to be a `generator` reference and what was specifically `generatedBy`.